src/core/hle/kernel/time_manager.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 yuzu Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <memory>
+
+#include "core/hle/kernel/object.h"
+
+namespace Core {
+class System;
+} // namespace Core
+
+namespace Core::Timing {
+struct EventType;
+} // namespace Core::Timing
+
+namespace Kernel {
+
+class Thread;
+
+class TimeManager {
+public:
+    TimeManager(Core::System& system);
```

Comment:
```suggestion
    explicit TimeManager(Core::System& system);
```

Destructor should also be placed into the cpp file as well.